src/coffee-script.coffee

```diff
@@ -63,6 +63,13 @@ exports.compile = compile = withPrettyErrors (code, options) ->
     token[1] for token in tokens when token[0] is 'IDENTIFIER'
   )
 
+  # Check for import or export; if found, force bare mode
+  unless options.bare? and options.bare is yes
+    for token in tokens
+      if token[0] in ['IMPORT', 'EXPORT']
+        options.bare = yes
```

Comment:
The wrapper function’s purpose is to [prevent accidental leaks into the global scope](http://coffeescript.org/#lexical-scope) when running in a browser. By definition a project with module imports is a multiple-file project, and something that will undergo further processing before a browser gets it. I don’t think we need the protection that the wrapper provides for such a project. Conversely, an `import` inside such a wrapper will fail to compile; and not all build chains provide easy ways to pass flags to the CoffeeScript compiler. I don’t want to make people scratch their heads trying to figure out how to pass flags to the compiler in Webpack/Gulp/Broccoli/Browserify/`ember-cli-coffeescript`/whatever, when the benefits of the safety wrapper are negligible or perhaps unnecessary in this case.